app/assets/stylesheets/application.css.scss

```diff
@@ -29,6 +29,18 @@
   color: red;
 }
 
+.formheader {
+  text-align: center
+}
+
+.new_sitcom {
+  width:985px; margin:0 auto
+}
+
+.edit_sitcom {
+  width:985px; margin:0 auto
```

Comment:
Properties should be ordered margin, width<br>Property 'margin' should be placed on own line<br>Colon after property should be followed by one space<br>Declaration should be terminated by a semicolon